test/net/NetTest.ooc

```diff
@@ -8,14 +8,73 @@
 
 use unit
 use net
+use concurrent
 
-//TODO: Write actual tests for source/net
-// Right now, this is just to make sure that the code is processed and compiled when testing
 NetTest: class extends Fixture {
 	init: func {
 		super("Net")
-		this add("No test", func {
-			expect(true)
+		this add("TCP Socket Client Server", func {
+			clientThread := Thread new(func {
+				clientIpText := t"0.0.0.0"
+				ip := IP4Address new(clientIpText toString())
+				clientMessageNumber := 0
+				clientSocket := TCPSocket new(SocketAddress new(ip, 8000))
+				clientSocket connect()
+				clientReceivedData: Int
+				clientSocket out write(clientMessageNumber as Char)
+				while (clientMessageNumber < 10) {
+					clientReceivedData = clientSocket in read() as Int
```

Comment:
`clientSocket in read()` - I don't recognize this syntax from anywhere else. What does it mean?